splice_access_api/src/main/java/com/splicemachine/access/configuration/OlapConfigurations.java

```diff
@@ -116,6 +124,27 @@
     public static final String OLAP_LOG4J_CONFIG = "splice.olap.log4j.configuration";
     public static final String DEFAULT_OLAP_LOG4J_CONFIG = null;
 
+    /* Map of roles to Splice queue names. Role names are case sensitive, typically they are all uppercase
+
+    Examples:
+        ISOLATEDROLE=customQueue
+        ROLE1=queue1,role2=queue2
+     */
+    public static final String OLAP_SERVER_ISOLATED_ROLES = "splice.olap_server.isolated.roles";
+    public static final String DEFAULT_OLAP_SERVER_ISOLATED_ROLES = "";
+
+    public static final String OLAP_SERVER_YARN_DEFAULT_QUEUE = "splice.olap_server.queue.default";
+    public static final String DEFAULT_OLAP_SERVER_YARN_DEFAULT_QUEUE = "default";
```

Comment:
Ah! So it is a constant the code could've referenced the name by (rather than having used the name directly). Why not to use it?